docs/index.md

```diff
@@ -1,4 +1,164 @@
 HCA DCP Query Service
 =====================
 
-[User Stories](USER_STORIES.md)
+# Introduction
+
+The HCA DCP Query Service provides an interface for scientists and developers to query metadata associated with
+experimental and analysis data stored in the [Human Cell Atlas](https://staging.data.humancellatlas.org/)
+[Data Coordination Platform](https://www.humancellatlas.org/data-sharing) (DCP). Metadata from the
+[DCP Data Store](https://github.com/HumanCellAtlas/data-store) are indexed and stored in an
+[AWS Aurora](https://aws.amazon.com/rds/aurora/) [PostgreSQL](https://www.postgresql.org/) database.
+
+# Query via the API
+Queries to the database can be sent over HTTP through the Query Service API
```

Comment:
Move this whole sentence "Queries to the database can be sent over HTTP through the Query Service API
[Swagger Documentation](https://query.data.humancellatlas.org/v1/ui/#/)
or via the [Query Builder](https://query.data.humancellatlas.org/)" to the introduction, since this section is entirely on querying via the API.

Might be useful to add a section on Querying via the Query Builder as well, with a brief description of the cards. Also, is the Query Builder subject to the same async user experience?

Reply:
`is the Query Builder subject to the same async user experience?` yes 
`Might be useful to add a section on Querying via the Query Builder as well, with a brief description of the cards.` @kislyuk do you have anything written for this elsewhere?

Reply:
No. This would be a good place to add that section.